pkg/utils/utils.go

```diff
@@ -154,27 +153,27 @@ func GetComponentImage(component string) string {
 
 	envVarName, ok := COMPONENT_IMAGES[component]
 	if !ok {
-		logrus.Errorf("Environment variable name mapping missing for component: %s", component)
+		log.Info("Environment variable name mapping missing for component:", "component", component)
 		return ""
 	}
 	imageTag := os.Getenv(envVarName)
 	if imageTag == "" {
-		logrus.Errorf("No image tag defined for component '%s' by environment variable '%s'", component, envVarName)
+		log.Info("No image tag defined for component by environment variable", "component", component, "envVarName", envVarName)
 	}
-	logrus.Debugf("Setting component image for '%s' to: '%s'", component, imageTag)
+	log.Info("Setting component image for to", "component", component, "imageTag", imageTag)
 	return imageTag
 }
 
 func GetFileContents(filePath string) []byte {
 
 	if filePath == "" {
-		logrus.Debug("Empty file path provided for retrieving file contents")
+		log.Info("Empty file path provided for retrieving file contents")
```

Comment:
@blockloop whats the philosophy here?  this strikes me as either a debug message (for devel), warn message (nothing to do about it as a user other then log bug), NO message because you cant do anything

Reply:
I think this whole function is bad golang practice, to begin with. It needs to return `[]byte, error`, and let the caller handle the error however it wants. I've changed the same function in EO in my WIP PR.